File: Data Structures & Algorithms/baseball-game/submission-0.cpp
class Solution {
   public:
    int calPoints(vector<string>& operations) {
        stack<int> st;

        for (string op : operations) {
            if (op == "+") {
                // record sum previous records
                int first = st.top();
                st.pop();
                int second = st.top();
                int third = first + second;
                st.push(first);
                st.push(third);
            } else if (op == "C") {
                // invalidate remove previous score
                st.pop();
            } else if (op == "D") {
                int first = st.top();
                st.push(first*2);

            } else {
                int num = std::stoi(op);
                st.push(num);
            }
        }
        int sum = 0;
        while (!st.empty()) {
            int temp = st.top();
            st.pop();
            sum += temp;
        }
        return sum;
    }
};

File: Data Structures & Algorithms/sort-an-array/submission-0.py
class Solution:
    def sortArray(self, nums: List[int]) -> List[int]:
        n = len(nums)
        # [2 , 4 , 3 , 8 , 1]
        for i in range(n):
            for j in range(n):
                if nums[j] > nums[i]:
                    temp =  nums[i]
                    nums[i] =  nums[j]
                    nums[j] = temp
                
        return nums
        